README.md

```diff
@@ -6,14 +6,13 @@
   - [🛠 Built With](#built-with)
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo (Not Microverse logo) at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍

Reply:
Yes sure, I have made a logo and shall use in all my projects.